docker/puppetdb/Dockerfile

```diff
@@ -8,7 +8,7 @@ ARG UBUNTU_CODENAME=bionic
 
 FROM ubuntu:18.04 as base
 
-ARG DUMB_INIT_VERSION="1.2.1"
+ARG DUMB_INIT_VERSION="1.2.2"
```

Comment:
Not related to your change, but.... why is this an `ARG` ?  Doesn't seem like something we'd ever set externally.

Reply:
There are 2 ways of doing a variable in a Dockerfile and ARG is the appropriate one for this context.

ARGs are for build-time variables (even if not specified as a `--build-arg` it's still an input and gives you one source of truth for that value in the Dockerfile). They are actually implemented as environment variables that don't show up in final image.

ENVs are for runtime variables - signals to end users that something that can be changed at launch time.